### PR TITLE
Add subscribe callback queue for TPSStore

### PR DESCRIPTION
**Pull request by dmytrozavgorodniy**

### CODE REVIEW

**Review by itsmepetrov — APPROVED** [review]
Looks good to merge

File: ios/TPSStore/EventEmitter.m [hunk 1]
```diff
@@ -27,6 +27,13 @@ @implementation EventEmitter
 @protocol EventEmitterListener <NSObject>
 @property BOOL once;
 @property id callback;
+
+#if OS_OBJECT_USE_OBJC
+@property (nonatomic, strong, nullable) dispatch_queue_t callbackQueue;
+#else
+@property (nonatomic, assign, nullable) dispatch_queue_t callbackQueue;
```
Comment (cybergrind) [inline ios/TPSStore/EventEmitter.m:34]: Maybe here we can skip preprocessor's if

**Review by cybergrind — CHANGES_REQUESTED** [review]
please check comment

File: ios/TPSStore/EventEmitter.m (hunk 1 above)
Comment (cybergrind) [reply ios/TPSStore/EventEmitter.m:34]: nevermind, missed difference in definitions

**Pull request merged**